bot/exts/evergreen/timed.py

```diff
@@ -0,0 +1,44 @@
+from copy import copy
+from time import perf_counter
+
+from discord import Message
+from discord.ext import commands
+
+
+class TimedCommands(commands.Cog):
+    """Time the command execution of a command."""
+
+    @staticmethod
+    async def create_execution_context(ctx: commands.Context, command: str) -> commands.Context:
+        """Get a new execution context for a command."""
+        msg: Message = copy(ctx.message)
+        msg.content = f"{ctx.prefix}{command}"
+
+        return await ctx.bot.get_context(msg)
+
+    @commands.command(name="timed", aliases=["t"])
```

Comment:
It could be worth adding a `time` alias, but I don't mind either way.
```suggestion
    @commands.command(name="timed", aliases=["time", "t"])
```